src/marks/auto.js

```diff
@@ -340,6 +345,10 @@ function isReducer(reduce) {
   return false;
 }
 
+function isUnderspecifiedReduce({value, reduce}) {
+  return value === undefined && reduce !== undefined && reduce !== "count";
+}
```

Comment:
Suggestion:

```suggestion
function isUnderspecifiedReduce({value, reduce}) {
  return value == null && reduce != null && /^count$/i.test(reduce);
}
```

This:

- Treats null and undefined as the same for _value_ and _reduce_
- Coerces _reduce_ to a string
- Does case-insensitive match equivalent to the keyword helper

Reply:
nice!! ive always wondered why you use regexes in this sort of helper, this is a good explanation